src/makeRobot.js

```diff
@@ -38,7 +38,61 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const newRobot = {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version}, `
+      + `wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goBack(value = 1) {
+      if (value >= 0) {
+        this.coords.y -= value;
+      }
+
+      return this;
+    },
+    goForward(value = 1) {
+      if (value >= 0) {
+        this.coords.y += value;
+      }
+
+      return this;
+    },
+    goRight(value = 1) {
```

Comment:
```suggestion
    },
    
    goForward(value = 1) {
      if (value >= 0) {
        this.coords.y += value;
      }

      return this;
    },
    
    goRight(value = 1) {
```
Don't forget to add empty lines between methods